tests/protos.rs

```diff
@@ -284,6 +284,82 @@ fn configure_builder(builder: Builder) -> Builder {
                 builder_custom_into!(read_consistency::Value, self.read_consistency),
             ),
             ("SearchBatchPoints.timeout", DEFAULT_OPTION),
+            // SearchPointGroups
+            ("SearchPointGroups.filter", DEFAULT_OPTION_INTO),
+            (
+                "SearchPointGroups.with_payload",
+                builder_custom_into!(with_payload_selector::SelectorOptions, self.with_payload),
+            ),
+            ("SearchPointGroups.params", DEFAULT_OPTION_INTO),
+            ("SearchPointGroups.score_threshold", DEFAULT_OPTION_INTO),
+            ("SearchPointGroups.vector_name", DEFAULT_OPTION_INTO),
+            (
+                "SearchPointGroups.with_vectors",
+                builder_custom_into!(with_vectors_selector::SelectorOptions, self.with_vectors),
+            ),
+            (
+                "SearchPointGroups.read_consistency",
+                builder_custom_into!(read_consistency::Value, self.read_consistency),
+            ),
+            ("SearchPointGroups.with_lookup", DEFAULT_OPTION_INTO),
+            ("SearchPointGroups.timeout", DEFAULT_OPTION_INTO),
+            ("SearchPointGroups.shard_key_selector", DEFAULT_OPTION_INTO),
+            ("SearchPointGroups.sparse_indices", DEFAULT_OPTION_INTO),
+            // WithLookup
+            (
+                "WithLookup.with_payload",
+                builder_custom_into!(with_payload_selector::SelectorOptions, self.with_payload),
+            ),
+            (
+                "WithLookup.with_vectors",
+                builder_custom_into!(with_vectors_selector::SelectorOptions, self.with_vectors),
+            ),
+            // DeletePoints
+            // DeletePoints exists twice in qdrant.rs. This interfers with this hook.
+            // TODO: Find a way to build or manually implement for DeletePoints
+            /*
```

Comment:
I already added a builder `DeletePoints` when I noticed there are multiple structs in qdrant.rs with this name. One directly in qdrant.rs and one in a sub-module. The tonic builder hook can't differentiate between them (at least not as far as I know) so it creates the same type+field attributes for both structs. However those structs need slightly different attributes or compiling will fail. I'll investigate into this further in a separate PR.

Reply:
I've seen issues like this before, it's a bit annoying.

Separating the module- and struct name with a `.` or `::` does not work I presume?

Reply:
It looks like there is a way to do that with `.` in the [documentation](https://docs.rs/prost-build/latest/prost_build/struct.Config.html#method.btree_map)
However this currently doesn't work well with the macros and the other logic so I left it open and wanted to investigate further in a separate PR.